### ISSUE TITLE
nickname not saved unless you hit "Done"

### ISSUE THREAD

**Issue by dholbert** [mozilla/MozStumbler#112]
tl;dr: You have to hit "Done" to actually save the nickname edits, which is non-obvious, since you can hide the keyboard and un-focus the field, and the edits look like they persist, until you quit & re-launch the app.

STR:
1. Open app
2. Tap nickname field & modify the content. (do _NOT_ hit "Done")
3. Hit the "Hide Keyboard" downarrow button (at bottom-left of keyboard on my Nexus4)
4. Optional: Start scanning, stop scanning, whatever.
5. quit app by triggering app switcher & swipe app offscreen
6. Re-launch app

EXPECTED RESULTS: My nickname should look the way it was when I quit the app.
ACTUAL RESULTS: My nickname edits weren't saved.

**Comment by flaki** [comment]
As I see things now there is no `Done` button any more, so this issue seems obsolete to me.

**Comment by cpeterso** [comment]
The `Done` button is on the virtual keyboard and the key layouts might be different for different keyboards and locales.

**Comment by flaki** [comment]
Yeah, actually (I think) I have experienced the issue myself later on today. Will try to investigate this, too.

**Comment by cpeterso** [comment]
Fixed by #135.
